src/lib/util/create-link.ts

```diff
@@ -2,10 +2,9 @@ import type { Participant } from '$lib/stores/participants-store';
 
 type linkParams = { participant: Participant; assignment: Participant };
 
-export function createLink(url: string, { participant, assignment }: linkParams): string {
+export function createLink(url: string, { participant, assignment }: linkParams, encode:Function = btoa): string {
```

Comment:
Yes, now we can pass any kind of encoder we want. Let's change the name to `encoder` please.